client/scripts/views/components/inline_thread_composer.ts

```diff
@@ -90,7 +91,7 @@ const LinkPost: m.Component<ILinkPostAttrs, ILinkPostState> = {
         placeholder: 'Add a description (optional)',
         tabindex: 2,
         theme: 'bubble',
-        editorNamespace: 'new-link-inline',
+        editorNamespace: `${app.activeId()}new-link-inline`,
```

Comment:
hyphen between activeId and new-link-inline?

Reply:
yup good catch. This code never gets used, right? But good to keep it clean in case it comes back